Source/Krypton Components/TestForm/TestForm.csproj

```diff
@@ -41,15 +41,6 @@
     </ItemGroup>
 
     <ItemGroup>
-        <Reference Include="Krypton.Docking">
```

Comment:
More than just the `HintPath`! How will testForm "Pull in" the correct project references now ?

Reply:
This it the full csproj, compiles and runs...

```xml
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>WinExe</OutputType>
        <!--<TargetFrameworks>net48;net6.0-windows</TargetFrameworks>-->
        <TargetFramework>net48</TargetFramework>
        <Nullable>enable</Nullable>
        <WarningLevel>8</WarningLevel> <!-- https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves-->
        <AnalysisLevel>latest</AnalysisLevel>
        <UseWindowsForms>true</UseWindowsForms>
        <LangVersion>preview</LangVersion>
    </PropertyGroup>

    <ItemGroup>
        <ProjectReference Include="..\Krypton.Docking\Krypton.Docking 2022.csproj" />
        <ProjectReference Include="..\Krypton.Navigator\Krypton.Navigator 2022.csproj" />
        <ProjectReference Include="..\Krypton.Ribbon\Krypton.Ribbon 2022.csproj" />
        <ProjectReference Include="..\Krypton.Toolkit\Krypton.Toolkit 2022.csproj" />
        <ProjectReference Include="..\Krypton.Workspace\Krypton.Workspace 2022.csproj" />
    </ItemGroup>

    <ItemGroup>
        <Compile Update="Properties\Resources.Designer.cs">
            <DesignTime>True</DesignTime>
            <AutoGen>True</AutoGen>
            <DependentUpon>Resources.resx</DependentUpon>
        </Compile>
    </ItemGroup>

    <ItemGroup>
        <EmbeddedResource Update="Properties\Resources.resx">
            <Generator>ResXFileCodeGenerator</Generator>
            <LastGenOutput>Resources.Designer.cs</LastGenOutput>
        </EmbeddedResource>
    </ItemGroup>

    <ItemGroup>
        <Content Include="invoices.xml">
            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
        </Content>
    </ItemGroup>

    <ItemGroup>
        <Reference Include="System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" Condition="$(TargetFramework.StartsWith('net4'))">
            <SpecificVersion>True</SpecificVersion>
            <Version>4.0.0.0</Version>
            <!-- Designers for the `TFMs != net4` are "Pulled in" via Visual studios ".nuget\packages\microsoft.windowsdesktop.app.ref" -->
        </Reference>
    </ItemGroup>

</Project>
```